public/app/features/plugins/admin/components/PluginDetailsAngularDeprecation.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { Alert } from '@grafana/ui';
+
+import { CatalogPlugin } from '../types';
+
+type Props = {
+  className?: string;
+  plugin: CatalogPlugin;
+};
+
+// An Alert showing information about Angular deprecation notice.
+// If the plugin does not use Angular (!plugin.angularDetected), it returns null.
+export function PluginDetailsAngularDeprecation({ plugin, className }: Props): React.ReactElement | null {
+  if (!plugin.angularDetected) {
+    return null;
+  }
+  return (
+    <Alert severity="warning" title="Angular plugin" className={className}>
+      <p>This plugin is using deprecated plugin APIs.</p>
```

Comment:
Similar to the tooltip, we need to expand this copy. Explain why is deprecated and what effects installing or enabling angular plugins have.